client/src/partials/templates/modals/findReference.modal.html

```diff
@@ -43,7 +40,7 @@
           data-button-patient-invoice
           ng-click="$ctrl.selectDocType('patient_invoice')">
```

Comment:
This isn't your code but in the future this should use an object map to avoid making mistakes with string literals. 

For example: 
```js
$ctrl.selectDocType($ctrl.DOCUMENT_TYPE.PATIENT_INVOICE)
```

Ideally this could then be defined in `bhConstants` to be shared across the entire application. 

```js 
{ 
  DOCUMENT_TYPE : { 
    PATIENT_INVOICE : 50
  }
}
```